bc_obps/registration/api/user_operator.py

```diff
@@ -390,7 +390,7 @@ def create_operator_and_user_operator(request, payload: UserOperatorOperatorIn):
     response={200: RequestAccessOut, codes_4xx: Message},
     url_name="update_operator_and_user_operator",
 )
-@authorize(["industry_user"], UserOperator.get_all_industry_user_operator_roles())
+@authorize(["industry_user"], ["admin"])
```

Comment:
As far as I remember, the first user who requests access gets an `admin` role, and everyone else who follows gets `pending`. We didn't build role assignment into MVP, so I think this change might block everyone except the first requester from accessing this endpoint. Approving a user_operator sets their status to `Approved` but doesn't change their role.

Reply:
This will be handled by #470 . I've moved up its priority in our backlog